applications/DamApplication/custom_elements/small_displacement_interface_element.cpp

```diff
@@ -77,6 +77,10 @@ int SmallDisplacementInterfaceElement<TDim,TNumNodes>::Check(const ProcessInfo&
     if ( DENSITY.Key() == 0 )
         KRATOS_THROW_ERROR( std::invalid_argument, "DENSITY has Key zero! (check if the application is correctly registered", "" )
 
+    //Interface variables
+    if ( INITIAL_JOINT_WIDTH.Key() == 0 || Prop.Has( INITIAL_JOINT_WIDTH ) == false || Prop[INITIAL_JOINT_WIDTH] < 0.0 )
```

Comment:
```suggestion
    if ( Prop.Has( INITIAL_JOINT_WIDTH ) == false || Prop[INITIAL_JOINT_WIDTH] < 0.0 )
```

I don't think we have to check for the key of a variable anymore. @roigcarlo ?

Reply:
Tbh I am not sure. @joaquinirazabal and @ipouplana are the maintainers, so if they believe that is necessary I trust them.

As I understand, this is only executed once so its not a performance issue anyway,

Reply:
nope the key check is no longer necessary, since like 6 years :sweat_smile: